src/lib/list/list.ts

```diff
@@ -119,6 +119,14 @@ export class MatListItem extends _MatListItemMixinBase implements AfterContentIn
               @Optional() private _navList: MatNavList) {
     super();
     this._isNavList = !!_navList;
+
+    // If no type attributed is specified for <button>, set it to "button".
+    // If a type attribute is already specified, do nothing.
+    let element: HTMLElement = this._getHostElement();
```

Comment:
Since `_getHostElement()` defines the return type of `HTMLElement`, you don't need to define the type for the `element` variable.